setup.py

```diff
@@ -97,7 +97,7 @@
     "isort @ git+git://github.com/timothycrosley/isort.git@e63ae06ec7d70b06df9e528357650281a3d3ec22#egg=isort",
     "flake8",
 ]
-extras["dev"] = extras["testing"] + extras["quality"] + ["mecab-python3<1", "scikit-learn", "tensorflow<=2.2", "torch"]
+extras["dev"] = extras["testing"] + extras["quality"] + ["fugashi>=1.0,<2.0", "ipadic", "scikit-learn", "tensorflow<=2.2", "torch"]
```

Comment:
Side note: the deps here are not with the same version tags as the deps up there.